taskcluster/kinds/fetch/toolchains.yml

```diff
@@ -49,3 +49,13 @@ cuda:
         sha256: 68699036c12d71adb9ad2799dce2ff070270fab4488b90920b9756ab3f52c41c
         size: 4245586997
         artifact-name: cuda-source.run
+
+
+cuda-11:
```

Comment:
Thought (no action needed): It would be nice for searchability if all the fetches were named something like "fetch-cuda-11" to help disambiguate between the keys. I often find myself confused looking in the taskgraph for what is what. For instance a few lines up is `preprocess` and `fetch-preprocess` would be a lot clearer on what it is.

Reply:
Unfortunately, this would result in the tasks being named things like `fetch-fetch-preprocess`. (Generally we see the kind directories as a sort of namespace, so they typically end up in the task names as well.)

Reply:
You could explicitly set `label: fetch-cuda-11` though, and that should cause the kind prepending to be skipped. But as usual it depends on what transforms are used.